test/e2e/telemetry_logs_analysis_test.go

```diff
@@ -152,12 +168,22 @@ var _ = Describe("Telemetry Components Error/Warning Logs Analysis", Label("tele
 						)),
 					)))),
 				))
-			}, time.Second*120, periodic.TelemetryInterval).Should(Succeed())
+			}, consistentlyTimeout, periodic.TelemetryInterval).Should(Succeed())
 		})
 
-		// TODO: Should not have any ERROR/WARNING level logs in the FluentBit containers
-		// TODO: configmap: FLuentBit, exclude_path (excluding self logs)
-		// telemetry-manager/blob/test/check-error-logs/internal/fluentbit/config/builder/input.go#L15-L16
+		It("Should not have any ERROR/WARNING logs in the FluentBit containers", func() {
+			Consistently(func(g Gomega) {
+				resp, err := proxyClient.Get(fluentBitLogTelemetryExportURL)
+				g.Expect(err).NotTo(HaveOccurred())
+				g.Expect(resp).To(HaveHTTPStatus(http.StatusOK))
+				g.Expect(resp).To(HaveHTTPBody(
+					Not(ContainLd(ContainLogRecord(SatisfyAll(
+						WithPodName(ContainSubstring("telemetry-")),
```

Comment:
Since you are searching for the same prefix as before, you just repeat the same test with the new mock backend. I suggest to be more specific about the prefix. Then you can also have specific whitelists per signal.

Reply:
I will see what I can do, although there are not too many components starting with "telemetry-...". It was more a way to test all our components together. But I guess the prefix condition can even be removed since it's not filtering too much out.

Reply:
I see that  you filter already on the container name in the LogPipeline. Then the prefix is actually not important here.  Let's keep it like this to be prepared for future changes.